### DISCUSSION TITLE
String Changes on Home Screen

### DISCUSSION THREAD

**Discussion by manjotsidhu** [PitchBlackRecoveryProject/android_bootable_recovery#66] [General]
## Community Discussion

Discussion for https://github.com/PitchBlackRecoveryProject/android_bootable_recovery/pull/65 and should we choose formal or slang.

### Open for All

This discussion is for all types of users, testers, maintainers and translators.

**Comment by manjotsidhu** [comment]
I think that Strings are really important because they have to explain the use case in just few words. Moreover, it should be tailored for all types of people who knows English very well and even the ones who can barely understand it. 
So in this sense, using the combination of formal and casual words should be the go-to according to me.

**Reply by LinkBoi00** [reply]
It's not that format to be honest

**Reply by rokibhasansagar** [reply]
Well, I remember telling Faraz to use slangs for-the-time-being as I couldn't think of any sub-headers 😆 
So, that's a bummer for me I guess. 😕

**Comment by LinkBoi00** [comment]
The main issue with the older ones is that the language is inconsistent, like I mentioned in #65. Random capital letters in the middle of a "sentence" for example.

**Reply by manjotsidhu** [reply]
That's definitely the issue, that's the reason why I am opening this discussion for making the best possible changes.

**Reply by DelightReza** [reply]
I Like Capital Letters In Each Words🤪

**Reply by rokibhasansagar** [reply]
For Card Titles and other page titles, capitalization on first characters of every word looks good.
But if they have sub-paragraph (like inside every cards), only sentence case is required.

**Reply by rokibhasansagar** [reply]
There are many words that needs to be fully/partially upper-cased.
Like, SELinux, SD card, ROM, PitchBlack (maybe), ADB, SHA2 -- these are needed to be expressed like they are now.
Without this form, they lack proper meaning.

**Comment by LinkBoi00** [comment]
I also think that the string for dm verity deactivation is not understandable, as is, it really doesn't reflect what it does

**Reply by manjotsidhu** [reply]
Suggest whatever changes you like, I will call some collaborators to give their votes/opinions as well on this thread.

**Reply by LinkBoi00** [reply]
That's what's cool about oss wew

**Reply by rokibhasansagar** [reply]
`dm-verity` is used for short understanding about ***Device-Mapper’s “verity” target***.
Disabling this usually helps to bypass Android Verified Boot and Encryption support.

I'm not sure how to explain this in short words, but I know what it does by the name of it.
That's why at least I don't  need extra explanation on that. But beginners might need though.

**Comment by manjotsidhu** [comment]
### Install Card String Changes

**Before** : Wanna Add changes in form of Zips or Img.
**Under Review/After**: Make modifications to your device. (by @LinkBoi00)

---
If the Review string sounds good, a 👍 reaction for it.
If you prefer the previous string, then a 👎 reaction for it.
If you have your own suggestions, drop a reply and we will vote on that.

**Reply by manjotsidhu** [reply]
@LinkBoi00 @AndroiableDroid @imranpopz @rezaadi0105 @md-raza @shovon668 and maintainers

**Reply by DelightReza** [reply]
Wut

**Reply by manjotsidhu** [reply]
> Wut

Vote on it

**Reply by imranpopz** [reply]
@LinkBoi00 thanks for your contribution and concern towards us,
I would like to stay on what we have before

And thanks once again @LinkBoi00

**Reply by manjotsidhu** [reply]
I actually don't like the previous string but I also think we can get a much better string than the new one.

**Reply by LinkBoi00** [reply]
> @LinkBoi00 thanks for your contribution and concern towards us,
> I would like to stay on what we have before
> 
> And thanks once again @LinkBoi00

You can close the PR then (#65)

**Reply by manjotsidhu** [reply]
> > @LinkBoi00 thanks for your contribution and concern towards us,
> > I would like to stay on what we have before
> > And thanks once again @LinkBoi00
> 
> You can close the PR then (#65)

Don't worry about that, that was just his opinion.

**Reply by manjotsidhu** [reply]
@AndroiableDroid @md-raza waiting for your opinion

**Reply by DelightReza** [reply]
Use ZIP or Image To Modify Your Device.

🤔

**Comment by IncorrigiblyBelligerent** [comment]
My suggestion would be to remove the helper strings and keep it clean and minimal. The only thing on the cards should be the name of the action in Uppercase.

**Reply by manjotsidhu** [reply]
The cards will look empty, if we remove content from it.

**Comment by manjotsidhu** [comment]
### Wipe Card Changes

**Before** : Release your device's Stress
**Under Review/After**: Wipe or format your device's data. (by @LinkBoi00)

---
If the Review string sounds good, a 👍 reaction for it.
If you prefer the previous string, then a 👎 reaction for it.
If you have your own suggestions, drop a reply and we will vote on that.

**Reply by MycroftHolmes1989** [reply]
Clean your device's data?

**Comment by manjotsidhu** [comment]
### Backup Card Changes

**Before** : Save Your Precious Gems at A Safer Place
**Under Review/After**: Save important data at a safer place (by @LinkBoi00)

---
If the Review string sounds good, a 👍 reaction for it.
If you prefer the previous string, then a 👎 reaction for it.
If you have your own suggestions, drop a reply and we will vote on that.

**Reply by MycroftHolmes1989** [reply]
"Make a copy of your important data." Saying "A Safer Place" may give them a false sense of security, because the backup is not inherently safe than the internal storage.

**Comment by manjotsidhu** [comment]
### Restore Card Changes

**Before** : Get back your Precious Gems. Have you saved?
**Under Review/After**: Restore data from previous backups. (by @LinkBoi00)

---
If the Review string sounds good, a 👍 reaction for it.
If you prefer the previous string, then a 👎 reaction for it.
If you have your own suggestions, drop a reply and we will vote on that.

**Comment by manjotsidhu** [comment]
### Mount Card Changes

**Before** : Get your device's Partitions Mounted
**Under Review/After**: Mount your device's partition. (by @LinkBoi00)

---
If the Review string sounds good, a 👍 reaction for it.
If you prefer the previous string, then a 👎 reaction for it.
If you have your own suggestions, drop a reply and we will vote on that.

**Reply by rokibhasansagar** [reply]
Well, the suggested line is more Modest that the previous one 😏

**Comment by manjotsidhu** [comment]
### Advanced Card Changes

**Before** : This page is not Enough for you? Switch to Advanced
**Under Review/After**: More options for advanced users (by @LinkBoi00) (edited)

---
If the Review string sounds good, a 👍 reaction for it.
If you prefer the previous string, then a 👎 reaction for it.
If you have your own suggestions, drop a reply and we will vote on that.

**Reply by rokibhasansagar** [reply]
More options for ~more~ advanced users
> Drop the second "more"

**Reply by manjotsidhu** [reply]
Changing Review String to "More options for advanced users"

**Reply by LinkBoi00** [reply]
I'll address feedback once all the strings have been "reviewed" fyi.
